src/DotVVM.Framework/Controls/SelectorBase.cs

```diff
@@ -26,32 +26,7 @@ public bool Enabled
         }
         public static readonly DotvvmProperty EnabledProperty =
             DotvvmPropertyWithFallback.Register<bool, SelectorBase>(nameof(Enabled),
-                FormControls.EnabledProperty, defaultPropertyInherit: true);
-        
-
-        /// <summary>
-        /// Gets or sets the name of property in the DataSource collection that will be displayed in the control.
-        /// </summary>
-        [Obsolete("Use ItemTextBinding property instead")]
-        public string DisplayMember
-        {
-            get { return (string)GetValue(DisplayMemberProperty); }
-            set { SetValue(DisplayMemberProperty, value); }
-        }
-        public static readonly DotvvmProperty DisplayMemberProperty =
-            DotvvmProperty.Register<string, SelectorBase>(t => t.DisplayMember, "");
-
-        /// <summary>
-        /// Gets or sets the name of property in the DataSource collection that will be passed to the SelectedValue property when the item is selected.
-        /// </summary>
-        [Obsolete("Use ItemValueBinding property instead")]
-        public string ValueMember
-        {
-            get { return (string)GetValue(ValueMemberProperty); }
-            set { SetValue(ValueMemberProperty, value); }
-        }
-        public static readonly DotvvmProperty ValueMemberProperty =
-            DotvvmProperty.Register<string, SelectorBase>(t => t.ValueMember, "");
```

Comment:
Also v2.0 ones. And these are also used in the markup, so we could not remove these anyway.

Reply:
Ok, we will remove these in 3.0. BTW: Shouldn't we also add `ItemKeyBinding` to fix #156?

Reply:
I don't like the `ItemKeyBinding` as it requires reimplementing knockout's `options` and `value` binding, doing a deep clone of the item (which may break validation errors IMO) and will match different items based on a ID, which may IMO cause even more problems. I'd at least wait for VDOM update with this one, as it removes at least some of the disadvatages.